### DISCUSSION TITLE
Styles order in css file

### DISCUSSION THREAD

**Discussion by ancashoria** [vanilla-extract-css/vanilla-extract#301] [Q&A] [answered]
Hi, I'm really loving vanilla-extract, it's awesome.

I'm trying to use the `<Box />` [component](https://github.com/seek-oss/vanilla-extract/blob/master/site/src/system/Box/Box.tsx#L57) from the website, but my reset style is overwriting my sprinkles styles because the reset styles are located at the end of the generated css file.

Can I control the order?
I see that on your website you are able to place the reset style [at the very top](https://vanilla-extract.style/main.css) of the generated css. How did you do this?

**Comment by mattcompiles** [comment, marked as answer]
Styles are ordered by your bundler. Most bundlers will order using the order your imports occur. In braid we expect users to import the reset before any themes or components to ensure its the first style in the file.

**Reply by ancashoria** [reply]
Thanks Matt, I'll look into it.
